packages/synapse-react-client/src/components/HasAccess/HasAccessV2.integration.test.tsx

```diff
@@ -93,43 +86,13 @@ describe('HasAccess tests', () => {
   })
 
   it('User has all permissions on a standard FileEntity and any Access Requirements have been met', async () => {
-    const entityBundle: EntityBundle = {
-      ...mockFileEntityBundle,
-      permissions: {
-        ...mockFileEntityBundle.permissions,
-        canDownload: true,
-      },
-    }
```

Comment:
This component no longer fetches the entity bundle, so test setup massively simplifies